typescript/src/languages/en-US/commands/management.json

```
@@ -357,21 +359,58 @@
 		"examples": [
 			"#member-logs",
 			"here"
+		],
+		"reminder": "All messages are in embeds so you will need to enable the permission **{{EMBED_LINKS, permissions}}** for Skyra."
+	},
+	"setMemberRemoveLogsDescription": "Set the member remove logs channel.",
+	"setMemberRemoveLogsExtended": {
+		"usages": [
+			"here/TextChannel"
+		],
+		"extendedHelp": "This command helps you setting up the member remove log channel, which will send messages each time a user leaves, is kicked, or banned.\nIn the case of the last two, it will send a special \"User Kicked\" or \"User Banned\" event.",
```

```suggestion
		"extendedHelp": "This command helps you setting up the member remove log channel, which will receive a message each time a user leaves, is kicked, or is banned.\nIn the case of the last two, it will send a special \"User Kicked\" or \"User Banned\" event.",
```

Didn't know channels were capable of sending messages